DigitalLearningSolutions.Web/Styles/layout.scss

```diff
@@ -343,3 +343,9 @@ nav, .nhsuk-header__navigation, #header-navigation {
     margin-top: 10px;
   }
 }
+
+@media (max-width: 48.0525em) {

```

Comment:
That max-width: 48.0525em seems oddly specific - can that be pulled out to a scss constant to give it some context maybe?

Reply:
This has been updated.

Reply:
Thanks @OluwatobiAwe Your 768px constant is a much more familar (non-magic) number thanks 👍